README.md

```diff
@@ -325,6 +325,27 @@ errors = [{ "title": "Invalid Attribute", "detail": "First name must contain at
 JSONAPI::Serializer.serialize_errors(errors)
 ```
 
+If you are using ActiveModel, you can pass in an object's errors:
```

Comment:
We should probably explicitly call out Rails models here (some people don't know that Rails models are ActiveModel's underneath). Like "If you are using Rails models or plain ActiveModel, you can..."